src/dispatch.rs

```diff
@@ -290,8 +294,8 @@ impl ApduDispatch
 
                 if self.was_request_chained || res.len() > interchanges::SIZE {
 
-                    // Send 256 bytes max at a time.
-                    let boundary = core::cmp::min(256, res.len());
+                    // Do not send more than the expcted bytes
```

Comment:
typo

Reply:
Also, please update the comment in line 284.  (I can’t comment on it directly as it’s not part of the diff.)